src/components/styles/Theme.tsx

```diff
@@ -0,0 +1,281 @@
+import { faCheckCircle, faExclamationCircle, faInfoCircle, faMinusCircle } from '@fortawesome/free-solid-svg-icons';
+import React, { FC } from 'react';
+import { colors } from '../../constants/colors';
+import { typography } from '../../constants/typography';
+import Icon from '../atoms/Icon/Icon';
+import { CardStyling } from '../organisms/Card/Card/Card';
+
+interface ButtonTheme {
+  text: string;
+  bg: string;
+  hover: string;
+}
+
+export type AlertLevel = 'info' | 'alert' | 'warning' | 'success' | 'brand';
+interface CardInfo {
+  headingSize: string;
+  textSize: string;
+  boxShadowStyle: string;
+  borderStyle: string;
+  backgroundStyle: string;
+}
+
+export interface AppTheme {
+  alert: Record<
```

Comment:
I'd break this up into named interfaces such as:

```
interface AlertTheme = Record<AlertLevel, { icon: string, background: string; ...etc } 
```

be easier to make individual changes then

Reply:
excellent plan